src/main/app/src/Components/Landscape/Search/Facets.tsx

```diff
@@ -50,7 +53,7 @@ const Facets: React.FC<FacetsProps> = ({ facets, addFacet, saveSearch }) => {
                 style={{ margin: 1, backgroundColor: cv.color }}
                 size={'small'}
                 key={facet.dim + '' + cv.label}
-                label={cv.label}
+                label={<MappedString mapKey={cv.label} />}
```

Comment:
![image](https://user-images.githubusercontent.com/143695/141929023-ac604fb3-12f7-4419-91d5-524caa92432d.png)

Please check why those values are not mapped although the mapping configuration exists.

Reply:
Not an error, I did not update the backend.